### DISCUSSION TITLE
Issues after Docker 26.1.3 update

### DISCUSSION THREAD

**Discussion by seal-82** [dani-garcia/vaultwarden#4574] [General]
Hi everybody, 

after update to Docker-CE 26.1.3 I ran into issues with my vaultwarden. In my setup behind a traefik proxy, it suddenly threw the error 502 bad gateway. After rollback to docker 26.1.2, everything is back to normal. 

Vaultwarden logs didnt show any errors.

Anybody else affected by this issue?

**Comment by BlackDex** [comment]
I'm running the same version, but do not seem to have an issue.
I would suggest to check the reverse proxy logs too, and the information of `docker ps` if that shows something.

**Reply by seal-82** [reply]
Thanks for your feedback. I checked traefik status website. There I did not get any errors. Will check all the logs and investigate further after my vacation. But maybe it is related to docker 26.1.3 in combination with traefik 3.0.0...

**Reply by BlackDex** [reply]
It could be that, or something changed in the networking. But that works be strange in a patch release of docker though

**Reply by seal-82** [reply]
Good to know I'm not the only one 😊 vaultwarden container also gives me status healthy and in traefik dashboard there is no error...

**Comment by TaisukeIto** [comment]
> Hi everybody,
> 
> after update to Docker-CE 26.1.3 I ran into issues with my vaultwarden. In my setup behind a traefik proxy, it suddenly threw the error 502 bad gateway. After rollback to docker 26.1.2, everything is back to normal.
> 
> Vaultwarden logs didnt show any errors.
> 
> Anybody else affected by this issue?

Me too.
after update to Docker-CE 26.1.3 I ran into issues

**Comment by clemone210** [comment]
I had this too.
I had to change the port to 80 instead of 88 which is exposed.
So I have it with Cloudflare Tunnel and before I had vaultwarden:88 in my tunnel config, after a change to vaultwarden:80 it works again.

**Comment by seal-82** [comment]
wanted to give you an update on the topic. After upgrade to current docker-sts version, access to vaultwarden is possible again. I'm not really sure what was the issue. But I'm happy to have vaultwarden working again with latest docker verion :-)
